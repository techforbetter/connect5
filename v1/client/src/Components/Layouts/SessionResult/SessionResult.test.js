import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter, Route } from "react-router-dom";

import SessionResults from "./index";

test("snapshot for 'SessionResults' layout", () => {
  const context = {};

  const component = renderer.create(
    <BrowserRouter context={context}>
      <Route path="/session/details/:sessionId/:sessionType" exact component={SessionResults} />
    </BrowserRouter>,
  );
  expect(component).toMatchSnapshot();
});
