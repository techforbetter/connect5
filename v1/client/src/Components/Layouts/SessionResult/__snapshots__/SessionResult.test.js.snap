// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for 'SessionResults' layout 1`] = `null`;
